contracts/periphery/Periphery.sol

```diff
@@ -162,6 +168,17 @@ contract Periphery is IPeriphery {
             _marginDelta = -_position.margin;
         }
 
+        if (msg.value > 0) {
+            if (address(_underlyingToken) == address(weth)) {
+                uint256 ethPassed = msg.value;
+
+                weth.deposit{value: msg.value}();
+                _underlyingToken.safeTransfer(msg.sender, ethPassed);
```

Comment:
So we are converting all provided ETH into WETH and sending it back to them? I guess this is nice in that it allows the rest of the code to remain unchanged. But it also potentially has the side effect of converting some of their ETH into WETH. Maybe that's OK. Is this what other protocols do?